### PR TITLE
KeyVault Certificates adding README and samples

### PR DESCRIPTION
**Pull request by schaabs**

### CODE REVIEW

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/samples/Sample1_HelloWorld.cs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            CertificateOperation newCertOp = client.StartCreateCertificate(certificate.Name, certificate.Policy);
+
+            while (!newCertOp.HasCompleted)
+            {
+                newCertOp.UpdateStatus();
+
+                Thread.Sleep(newCertOp.PollingInterval);
+            }
+
+            // The certificate is no longer needed, need to delete it from the Key Vault.
```
Comment (heaths) [inline sdk/keyvault/Azure.Security.KeyVault.Certificates/samples/Sample1_HelloWorld.cs]: Two separate sentences.

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/samples/Sample3_GetKeysAsync.cs [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    /// <summary>
+    /// Sample demonstrates how to list keys and versions of a given key,
+    /// and list deleted keys in a soft-delete enabled Key Vault
+    /// using the asynchronous methods of the KeyClient.
+    /// </summary>
+    [LiveOnly]
+    public partial class GetCertificates
+    {
+        [Test]
+        public async Task GetKeysAsync()
```
Comment (heaths) [inline sdk/keyvault/Azure.Security.KeyVault.Certificates/samples/Sample3_GetKeysAsync.cs]: `GetCertificatesAsync`?

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
+# Azure Key Vault Certificate client library for .NET
+Azure Key Vault is a cloud service that provides secure storage and automated management of certificates used throughout a cloud application. Multiple certificate, and multiple versions of the same certificate, can be kept in the Key Vault. Each certificate in the vault has a policy associated with it which controls the issuance and lifetime of the certificate, along with actions to be taken as certificates near expiry.
+
+The Azure Key Vault Certificate client library enables programmatically managing certificates, offering methods to create, update, list and delete certificates, policies, issuers and contacts. The library also supports managing pending certificate operations, and management of deleted certificates.
```
Comment (heaths) [inline sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md]: Nit: I recommend the Oxford comma everywhere for readability. It helps separate clauses, which is especially important with complex list statements such as this.

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+    ```
+    > --key-permissions:
+    > Accepted values: backup, create, decrypt, delete, encrypt, get, import, list, purge, recover, restore, sign, unwrapKey, update, verify, wrapKey
+
+* Use the above mentioned Key Vault name to retrieve details of your Vault which also contains your Key Vault URL:
+    ```PowerShell
+    az keyvault show --name <your-key-vault-name> 
+    ```
+
+#### Create KeyClient
```
Comment (heaths) [inline sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md]: `CertificateClient`

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+* [Create a Certificate](#create-a-certificate)
+* [Retrieve a Certificate](#retrieve-a-certificate)
+* [Update an existing Certificate](#update-an-existing-certificate)
+* [Delete a Certificate](#delete-a-certificate)
+* [List Certificates](#list-certificates)
+
+### Create a Certificate
+`StartCreateCertificate` creates a Certificate to be stored in the Azure Key Vault. If a certificate with 
+the same name already exists, then a new version of the certificate is created.
+When creating the certificate the user can specify the policy which controls the certificate lifetime. If no policy is speicired the default policy will be used. The `StartCreateCertificate` operation returns a `CertificateOperation`. The following example creates a self signed certificate with the default policy.
```
Comment (heaths) [inline sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md]: "specified"

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+### Update an existing Certificate
+`UpdateCertificate` updates a certificate stored in the Key Vault.
+```c#
+IDictionary<string, string> tags = new Dictionary<string, string>() { { "key1", "value1" } };
+
+Certificate updated = await Client.UpdateCertificateAsync(certName, certVersion, tags: expTags);
+```
+
+### Delete a Certificate
+`delete_certificate` deletes all versions of a certificate stored in the Key Vault. When [soft-delete][soft_delete]
```
Comment (heaths) [inline sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md]: `DeleteCertificate`

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md [hunk 7]
```diff
@@ -0,0 +1,214 @@
...
+
+Console.WriteLine(deletedCert.ScheduledPurgeDate);
+
+await Client.PurgeDeletedCertificate(certName);
+```
+
+## Troubleshooting
+
+### General
+When you interact with the Azure Key Vault Key client library using the .NET SDK, errors returned by the service correspond to the same HTTP status codes returned for [REST API][keyvault_rest] requests.
```
Comment (heaths) [inline sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md]: "Certificate Client"

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md [hunk 8]
```diff
@@ -0,0 +1,214 @@
...
+
+For example, if you try to retrieve a Key that doesn't exist in your Key Vault, a `404` error is returned, indicating `Not Found`.
```
Comment (heaths) [inline sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md]: "Certificate". I'll not repeat this comment (may be more). I'm seeing a lot of (copy and paste?) references to Key Client and Keys.

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md [hunk 9]
```diff
@@ -0,0 +1,214 @@
...
+    Strict-Transport-Security: max-age=31536000;includeSubDomains
+    X-Content-Type-Options: nosniff
+    Date: Tue, 18 Jun 2019 16:02:11 GMT
+    Content-Length: 75
+    Content-Type: application/json; charset=utf-8
+    Expires: -1
+```
+## Next steps
+Key Vault Certificates client library samples are available to you in this GitHub repository. These samples provide example code for additional scenarios commonly encountered while working with Key Vault:
+* [HelloWorld.cs][hello_world_sync] and [HelloWorldAsync.cs][hello_world_async] - for working with Azure Key Vault, including:
```
Comment (heaths) [inline sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md]: These links aren't defined below.

**Review by heaths — APPROVED** [review]

**Comment by schaabs** [comment]
/azp run net - keyvault - ci

Rerunning pipeline due to issue https://github.com/Azure/azure-sdk-for-net/issues/7604

**Pull request merged**